internal/controller/controllers/infraenv_controller.go

```diff
@@ -406,6 +418,7 @@ func CreateInfraEnvParams(infraEnv *aiv1beta1.InfraEnv, imageType models.ImageTy
 			CPUArchitecture:        infraEnv.Spec.CpuArchitecture,
 			ClusterID:              clusterID,
 			OpenshiftVersion:       openshiftVersion,
+			AdditionalTrustBundle:  additionalTrustBundle,
```

Comment:
Why is this `additionalTrustBundle` not coming from `infraEnv.Spec.AdditionalTrustBundle`?